buttons/button_base.h

```diff
@@ -24,20 +24,29 @@ class ButtonBase : public Looper,
       while (!DebouncedRead()) YIELD();
       prop.Event(button_, EVENT_PRESSED);
       if (millis() - push_millis_ < 500) {
-        prop.Event(button_, EVENT_DOUBLE_CLICK);
+        doubleclicked_ = true;
       } else {
         push_millis_ = millis();
         current_modifiers |= button_;
       }
       while (DebouncedRead()) {
         if (millis() - push_millis_ > 300) {
-          prop.Event(button_, EVENT_HELD);
+          if (!doubleclicked_) {
+            prop.Event(button_, EVENT_HELD);
+          } else {
+            prop.Event(button_, EVENT_DOUBLE_CLICK_HELD);
```

Comment:
if we're going to have EVENT_DOUBLE_CLICK_HELD, why not add DOUBLE_CLICK_EVENT_HELD_MEDIUM, DOUBLE_CLICK_EVENT_HELD_LONG while we are at it?

Reply:
done